src/article_maker.cc

```diff
@@ -63,14 +65,19 @@ std::string ArticleMaker::makeHtmlHeader( QString const & word, QString const &
   // document ready ,init webchannel
   {
     result += R"(
-    <script>
-     $_$(document).ready( function ($){ 
-         console.log("webchannel ready..."); 
-         new QWebChannel(qt.webChannelTransport, function(channel) { 
-             window.articleview = channel.objects.articleview; 
-       }); 
-     }); 
-    </script>
+<script>
+  function gd_init_QtWebChannel(){
+   new QWebChannel(qt.webChannelTransport, function(channel) {
+         window.articleview = channel.objects.articleview;});
+   console.log("Qt webchannel ready...");
+  };
+
+  if (document.readyState !== "loading") {
+    gd_init_QtWebChannel();
+  } else {
+    document.addEventListener("DOMContentLoaded", gd_init_QtWebChannel);
+  };
+</script>
```

Comment:
I think we should move these from `.cc` to `.js`. I will probably do it in another PR.